QUICKSTART-VMWARE.md

```diff
@@ -0,0 +1,248 @@
+
+#### Bootstrap token
+Nodes require a token to establish trust between the node and the control plane.
+Tokens must be [used within 24 hours](https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-token/), but once the node has booted and registered with the cluster, it isn't used again.
+```
+kubeadm --kubeconfig <your kubeconfig> token create
+```
+
+#### Cluster Certificate
+This is the base64 encoded cluster certificate authority.
+```
+kubectl --kubeconfig=<your kubeconfig> config view --raw -o json | jq -r '.clusters[0].cluster."certificate-authority-data"'
+```
+
+## Configuring Bottlerocket
+
+In order to join Bottlerocket to your cluster, it must be configured via user data.
+There are multiple methods of passing user data to Bottlerocket in VMware; we will demonstrate all of them.
+
+Create a file called `userdata.toml` and populate it with the values you just retrieved.
+```
+[settings.kubernetes]
+api-server = "<value here>"
+cluster-dns-ip = "<value here>"
+bootstrap-token = "<value here>"
+cluster-certificate = "<value here>"
+```
+
+### Accessing your Bottlerocket guest via host containers
+For remote access to your running Bottlerocket VMs, you will need to add additional user data to enable host containers.
+The Bottlerocket VMware images don't include any host containers enabled by default.
+But don't worry!
+You can use our [admin](https://github.com/bottlerocket-os/bottlerocket-admin-container) and/or [control](https://github.com/bottlerocket-os/bottlerocket-control-container) containers, they just need to be configured first.
+
+#### Admin container
+If you would like to use the admin container, you will need to create some base64 encoded user data which will be passed to the container at runtime.
+Details for the required host container user data are covered in the [admin container documentation](https://github.com/bottlerocket-os/bottlerocket-admin-container#authenticating-with-the-admin-container).
+Once you've created the required admin container user data, you can enable the admin container by adding the following to your Bottlerocket user data:
+```
+[settings.host-containers.admin]
+enabled=true
+user-data = "<YOUR USER DATA>"
+```
+
+#### Control container
+Enabling the control container is very similar to the admin container; you will create some base64 encoded user data that will be passed to the container at runtime.
+This user data includes an activation ID and code that are detailed in the [control container documentation](https://github.com/bottlerocket-os/bottlerocket-control-container#connecting-to-aws-systems-manager-ssm).
+Once you've created the required control container user data, you can enable it by adding the following to your Bottlerocket user data:
+```
+[settings.host-containers.control]
+enabled = true
+user-data= "<YOUR USER DATA>"
+```
+
+### **PREFERRED**: Setting user data via "guestinfo" interface
+> Note: You must set these values *before* you start the VM for the first time.
```

Comment:
Is there a way we can make this stand out more? I'm worried folks skimming will miss it.